tests/test.c

```diff
@@ -116,6 +116,12 @@ child(int master, int slave)
 	if (ioctl(slave, TIOCSCTTY, NULL) < 0)
 		err(1, "ioctl");
 
+	/* Enable malloc.conf(5) options on OpenBSD which will abort the pick
+	 * process when reading/writing out-of-bounds or accessing already freed
+	 * memory. */
+	if (setenv("MALLOC_OPTIONS", "S", 0) < 0)
```

Comment:
This can alternatively be done with the global `malloc_options` char pointer; see [`usr.sbin/httpd/patterns/patterns-tester.c`](http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/regress/usr.sbin/httpd/patterns/patterns-tester.c?rev=1.1&content-type=text/x-cvsweb-markup) for an example (lines 25 and 66).